ansible/roles/ooni-collector/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+# FIXME: do we _really_ need nginx in front of the collector?
```

Comment:
**Question 1**: do we _really_ need nginx in front of the collector?

Reply:
I would remove it for the time being as we haven't tried the deployment this way and stuff may break.

Reply:
I agree